frontend/src/components/navigation/ToggleButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./ToggleButton.css";
+
+export type ToggleButtonProps = {
+  currentState: boolean;
+  trueStateName: string;
+  falseStateName: string;
+  onToggle: (newState: boolean) => void;
+};
+
+const ToggleButton = ({
+  currentState,
+  trueStateName,
+  falseStateName,
+  onToggle,
+}: ToggleButtonProps) => {
+  /**
+   * How can we make it super visible that the true state is always displayed on the left
```

Comment:
Rather than true vs false (state) we can think of primary vs secondary (state), where left is default selected. left vs right wording may also be clear. Open to thoughts on naming. 

`trueStateName` does feel like an odd name for a toggle button label. What do you think of `primaryToggleName`?

Reply:
how about `LeftToggleLabel`?

Reply:
`LeftToggleLabel` sounds good